### ISSUE TITLE
Doesn't work after updating to React 16.5

### ISSUE THREAD

**Issue by dmlukichev** [ethanselzer/react-image-magnify#47]
```
Warning: The <ReactCursorPosition /> component appears to have a render method, but doesn't extend React.Component. This is likely to cause errors. Change ReactCursorPosition to extend React.Component instead.
Uncaught TypeError: Cannot call a class as a function at _classCallCheck (ReactCursorPosition.js:3)
```

Problem lies with more strict rules for class call check in latest react version. Because of this old versions of `react-cursor-position` stopped working work.
They already released [new version](https://github.com/ethanselzer/react-cursor-position/pull/25) with fix, but since `react-image-magnify` uses old one we still have this problem.
Updating dependency will solve this issue.

I hope it will be closed pretty quickly.
Otherwise you can resolve it temporary by directly editing `package-lock.json` and setting proper version there.
```
"react-image-magnify": {
      "requires": {
        /* ... */
        "react-cursor-position": "3.0.2",
      },
      "dependencies": {
        /* ... */
        "react-cursor-position": {
          "#todo@dmlukichev": "Remove this after react-image-magnify will update their deps",
          "version": "3.0.2",
          "resolved": "https://registry.npmjs.org/react-cursor-position/-/react-cursor-position-3.0.2.tgz",
          "integrity": "sha512-x6hv5FU9Uo3r8LVcIoF/9V0lLujMFTfvizUO3ExYg/o3+KHv2tGxuq+f4PcurV6AoVPRNFaiXps9ArUNGHAZTA==",
          "requires": {
            "object-assign": "^4.1.1",
            "object.omit": "^3.0.0",
            "prop-types": "^15.6.0"
          }
        }
      }
    },
```

**Comment by ethanselzer** [comment]
@dmlukichev - Thanks for identifying this issue! A fix is being worked on now. I will post here when it is ready.

**Comment by ethanselzer** [comment]
I think this issue is corrected in version 2.7.1. Please see this [CodeSandbox](https://codesandbox.io/s/6w0jxnjjvz) for verification. Thanks!

Closing as fixed by #48. Please reopen if needed.
